overlays/bootstrap.nix

```diff
@@ -903,8 +903,9 @@ in {
       (final.haskell-nix.hackage-package ({pkgs, ...}: {
         evalPackages = pkgs.buildPackages;
         name = "cabal-install";
-        version = "3.6.2.0";
+        version = "3.8.1.0";
         index-state = final.haskell-nix.internalHackageIndexState;
+        configureArgs = "--disable-benchmarks --disable-tests";
```

Comment:
I wonder if we should put this in the hackage quirks? It'll hit other people too. I had to do something like this to update cabal for plutus since the test suites are broken.

Reply:
I've switched it over to use the `haskell.nix.tool` function.  The `haskell.nix.tool` function knows we don't want a test or benchmark from the `hackage-project` and [already disables them by default](https://github.com/input-output-hk/haskell.nix/blob/2b9b99a537327df4dfaa93d7827394c5388bceb7/overlays/tools.nix#L68-L69).